CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+- minor updated for payment-novaramedia.com
+- added margin m and padding p
+- added space maker classes to media queries
```

Comment:
This doc will need an existing line for version 0.9.0 that just can say something like `Init library from base` or whatever. Then that will allow the (future) automation flow with the Unreleased section. I suppose also we could not care and just when you do the releasing flow on the main branch once mastered you just update this doc and we start writing the changelog from v0.10.0